Sources/Datadog/Core/FeaturesConfiguration.swift

```diff
@@ -82,6 +82,33 @@ extension FeaturesConfiguration {
         var rum: RUM?
         var urlSessionAutoInstrumentation: URLSessionAutoInstrumentation?
 
+        var logsEndpoint = configuration.logsEndpoint
```

Comment:
Hmmm...I guess we should add a task also in Android to align the APIs. @xgouchet what do you think ?

Reply:
FTR: It's already there in the Android SDK 👌.